doc/manuals/admin/database_model.md

```diff
@@ -254,39 +254,39 @@ Fields:
 Example document:
 
 ```
- {
```

Comment:
JSON hasn't changed actually, but for some reason github shows this as change...

NTC